File: Prob0001.java
import java.util.Scanner;

class Prob0001
{
    public static void main(String[] args) {
        int num = 0;
        int addn = 0;
        Scanner sc = new Scanner(System.in);
        
        System.out.print("Enter range(n):\t");
        num = sc.nextInt();
        
        // Prob1 pr1 = new Prob1(); 
        for(int i = 0; i < num; i++)
        {
            if(i % 3 == 0 | i % 5 == 0)
            {
                addn += i;
            }
        }
        System.out.println("Output is:" + addn);
        sc.close();
    }
}

File: Prob0001.c
#include <stdio.h>

void main()
{
    int num = 0, subt = 0, addn = 0;
    printf("Enter range(n):\t");
    scanf("%d", &num);

    //my Solution
    /*
    int i = 3;
    while(i < num)
    {
        if(i%15 == 0)
        {
            subt += i;
        }
        addn += i;
        i += 3;
    }
    
    i = 5;
    while(i < num)
    {
        addn += i;
        i += 5;
    }
    addn -= subt;
    printf("%d", addn);
    */
   
   //better approach
   for(int i = 0; i < num; i++)
   {
       if((i % 3 == 0 )||(i % 5 == 0))
       {
           addn += i;
       }
   }
   printf("%d", addn);
}

